tests/openvino/test_quantization.py

```diff
@@ -75,7 +75,7 @@
 class OVQuantizerTest(unittest.TestCase):
     SUPPORTED_ARCHITECTURES_WITH_EXPECTED_QUANTIZED_MATMULS = (
         (OVModelForSequenceClassification, "bert", 32, 35),
-        # (OVModelForCausalLM, "gpt2", 41, 23),
+        (OVModelForCausalLM, "gpt2", 31, 3),
```

Comment:
For some reason, `auto` and `ov` tests result in a different number of FQ nodes: 41 and 31 respectively. I'll take a look why.

Reply:
thanks for checking ! I updated it in https://github.com/huggingface/optimum-intel/pull/727/commits/054c66aa596415ec857f11691d394eb85686235e in the meantime

Reply:
yes makes sense, added it in https://github.com/huggingface/optimum-intel/pull/727/commits/d3223d053296d262fb63c49e0ddecea076aa7f05